src/axom/sidre/tests/sidre_view.cpp

```diff
@@ -1852,6 +1852,119 @@ TEST(sidre_view, deep_copy_shape)
 
 //------------------------------------------------------------------------------
 
+TEST(sidre_view, reshape_array)
+{
+  using namespace axom;
+  DataStore ds;
+  Group* root = ds.getRoot();
+
+  constexpr int DMAX = 4;
+  IndexType shapeOutput[DMAX];
+  int nDim = -1;
+
+  // A view with array in a buffer.
+  auto* viewA =
+    root->createView("viewA", sidre::detail::SidreTT<int32_t>::id, 12);
+
+  {
+    nDim = viewA->getShape(DMAX, shapeOutput);
+    EXPECT_EQ(viewA->getNumElements(), 12);
+    EXPECT_EQ(nDim, 1);
+    EXPECT_EQ(shapeOutput[0], 12);
+  }
+
```

Comment:
Would it make sense to add some tests w/ 1D versions of `reshapeArray` and `N x 1 x 1` versions
E.g.
```
reshapeArray(1,12);
reshapeArray(3, [12,1,1]);
```
in cases where the dimension was already 1 and in cases where it was different?

Reply:
Sure.  Done.